ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml

```diff
@@ -156,23 +156,66 @@ tests:
     - ref: ipi-install-heterogeneous
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-azure-ipi
-- as: azure-ipi-baselinecaps-v412-f28
```

Comment:
The job already got updated in https://github.com/openshift/release/pull/43642, so you can add your dedicated jobs for vset coverage, and leave this one untouched.

Reply:
I synced up the latest code, and it seems that this job is not touched in 4.14/4.15 multi-nightly.yaml in PR 43642, do we still need to keep this job or updated to vset job in this PR?

Reply:
Ah, sorry, another coming [PR](https://github.com/openshift/release/pull/43676) would resolve it.

Reply:
got it, fixed.